security/advisories/2017-SV4-guest-user/index.html

```diff
@@ -22,7 +22,7 @@ <h3 class="secvul-heading"><i class="fa fa-history"></i> Background</h3>
             <div class="small-12 medium-9 columns">
                 <p class="secvul-description">A "guest" user is enabled by default on all OMERO systems to allow clients to request non-sensitive information on startup e.g. the current server version. A few commands failed to check for the guest user and therefore allowed user-like actions. This included current sessions requests which allowed "bin/omero sessions who" to disclose information about when other users were logged in. Additionally,  "bin/omero script upload" would permit the upload of files.
                 </p>
-                <p class="secvul-description">A review was performed of all API methods, disabling all but the bare minimum of calls that are needed by "guest". Future versions may loosen the restriction to re-allow some minimal actions by "guest".
+                <p class="secvul-description">A review was performed of all API methods, disabling all but the bare minimum of calls that are needed by "guest". Future versions may loosen the restriction to re-allow some minimal actions by "guest". Note that this will also disable the password reset functionality in OMERO.web.
```

Comment:
Might be clear to put this after "The fix..." and say "This has also disabled the password reset functionality in OMERO.web"

Reply:
Admins *can* still reset others' passwords, I think it's just ordinary users can't use the "forgotten password" thing.

Reply:
@mtbc  - how does https://github.com/openmicroscopy/www.openmicroscopy.org/pull/144/commits/58032c377a4d8127724f580f8d92c922d97c6fc1 read?

Reply:
Certainly eliminates the ambiguity for me, thank you.